examples/complete/provider.tf

```diff
@@ -0,0 +1,47 @@
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
+}
+
+# used by the restapi provider to authenticate the API call based on API key
+data "ibm_iam_auth_token" "token_data" {
+}
+
+provider "restapi" {
+  uri                   = "https:"
+  alias                 = "kp"
+  write_returns_object  = false
+  create_returns_object = false
+  debug                 = false # set to true to show detailed logs, but use carefully as it might print sensitive values.
+  headers = {
+    Authorization    = data.ibm_iam_auth_token.token_data.iam_access_token
+    Bluemix-Instance = module.key_protect_all_inclusive.key_protect_guid
+    Content-Type     = "application/vnd.ibm.kms.policy+json"
+  }
+}
+
+
+provider "restapi" {
```

Comment:
Do we need this in the example if we dont have Secrets Manager?

Reply:
I need to check this I thought it was for kepyprotect but now not sure